koku/reporting/provider/ocp/models.py

```diff
@@ -323,6 +331,9 @@ class Meta:
     usage_start = models.DateTimeField(null=False)
     usage_end = models.DateTimeField(null=False)
 
+    pod_label_key = models.CharField(max_length=253, null=True)
+    pod_label_value = models.CharField(max_length=253, null=True)
```

Comment:
What led you to this max length?

Reply:
Most kubernetes objects have that max length built in. I am just kind of assuming labels are the same, but we could lift this.

Reply:
Was just hoping it was doc related and wondering if you had dug up any links in case we needed it in the future.